frontend/pages/_document.js

```diff
@@ -0,0 +1,27 @@
+import Document from 'next/document'
```

Comment:
Is there a reason this isn't a ts file?

Reply:
I think it's worth converting to typescript. If you look the other _app is a ts file too, and I think it's important for consistency

Reply:
SG, I'll change it into typescript!